app/src/main/kotlin/com/metrolist/music/widget/PlaylistWidgetReceiver.kt

```
@@ -0,0 +1,105 @@
+    override fun onReceive(context: Context, intent: Intent) {
+        super.onReceive(context, intent)
+
+        when (intent.action) {
+            ACTION_PLAY_TARGET -> {
+                val serviceIntent = Intent(context, MusicService::class.java).apply {
+                    action = ACTION_PLAY_TARGET
+                    putExtras(intent)
+                }
+                try {
+                    context.startService(serviceIntent)
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)